### PR TITLE
[MOB-2985]: Loyalty points user's balance

### PR DESCRIPTION
**Pull request by nurito-burrito**
## JIRA
[MOB-2985](https://karhoo.atlassian.net/browse/MOB-2985)

## DESCRIPTION
Loyalty points user's balance

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix 
- [ ] New feature 
- [ ] Tech Debt
- [ ] Breaking change 
- [ ] Documentation Update
- [x] Other: NSDK improvement

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [x] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [x] Lint and unit tests pass locally (if appropriate)
- [ ] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
[Add screenshots if any for changes you have made to the App/UI]

### CODE REVIEW

File: KarhooSDK/Service/Payment/KarhooPaymentService.swift [hunk 1]
```diff
@@ -39,13 +39,11 @@ final class KarhooPaymentService: PaymentService {
 
     func initialisePaymentSDK(paymentSDKTokenPayload: PaymentSDKTokenPayload) -> Call<PaymentSDKToken> {
         paymentSDKTokenInteractor.set(payload: paymentSDKTokenPayload)
-
```
Comment (EdwardW2) [inline KarhooSDK/Service/Payment/KarhooPaymentService.swift:42]: Why remove these spaces?

File: KarhooSDKIntegrationTests/Service/Loyalty/LoyaltyBalanceMethodSpec.swift [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            XCTAssertEqual(true, result.successValue()?.burnable)
+            XCTAssertEqual(10, result.successValue()?.points)
+            expectation.fulfill()
+        })
+
+        waitForExpectations(timeout: 1)
+    }
+    
+    /**
+      * Given: Requesting a cancellation fee
```
Comment (EdwardW2) [inline KarhooSDKIntegrationTests/Service/Loyalty/LoyaltyBalanceMethodSpec.swift:45]: Comment needs to be updated

File: KarhooSDK/Service/Payment/KarhooPaymentService.swift (hunk 1 above)
Comment (nurito-burrito) [reply KarhooSDK/Service/Payment/KarhooPaymentService.swift:42]: because it was supposed to be removed like all the other funcs in that class

**Review by EdwardW2 — APPROVED** [review]

File: KarhooSDKIntegrationTests/Service/Loyalty/LoyaltyBalanceMethodSpec.swift (hunk 2 above)
Comment (nurito-burrito) [reply KarhooSDKIntegrationTests/Service/Loyalty/LoyaltyBalanceMethodSpec.swift]: thanks, fixed

**Review by EdwardW2 — APPROVED** [review]

**Pull request merged**